src/content/docs/it/concepts/why-astro.mdx

```diff
@@ -0,0 +1,76 @@
+- 850 ms più veloce → 7% in più di conversioni ([COOK](https://web.dev/why-speed-matters/))
+- Ogni 1 secondo più lento → 10% in meno di utenti ([BBC](https://www.creativebloq.com/features/how-the-bbc-builds-websites-that-scale))
+
+In molti web framework, è facile creare un sito web che sembra ottimo durante lo sviluppo, ma che si carica lentamente una volta distribuito. Spesso il colpevole è JavaScript, poiché i telefoni degli utenti e i dispositivi meno potenti raramente corrispondono alla velocità di un computer portatile dello sviluppatore.
+
+La magia di Astro sta nel modo in cui combina i due valori spiegati sopra: un'attenzione al contenuto con un'architettura MPA orientata al server, per effettuare compromessi e offrire funzionalità che altri framework non possono offrire. Il risultato sono prestazioni web straordinarie per ogni sito web, direttamente dalla confezione. Il nostro obiettivo: **Dovrebbe essere quasi impossibile creare un sito web lento con Astro.**
+
+Un sito web Astro può [caricarsi più velocemente del 40% con il 90% in meno di JavaScript](https://twitter.com/t3dotgg/status/1437195415439360003) rispetto allo stesso sito costruito con il web framework React più popolare. Ma non prenderci solo a parole: guarda le prestazioni di Astro lasciate da Ryan Carniato (creatore di Solid.js e Marko) [senza parole](https://youtu.be/2ZEMb_H-LYE?t=8163).
+
+## Facile da usare
+
+**L'obiettivo di Astro è essere accessibile a ogni sviluppatore web.** Astro è stato progettato per sembrare familiare e accessibile indipendentemente dal livello di competenza o dall'esperienza passata nello sviluppo web.
+
+Abbiamo iniziato assicurandoci che tu possa utilizzare qualsiasi linguaggio di componenti UI che già conosci. React, Preact, Svelte, Vue, Solid, Lit e diversi altri sono tutti supportati per la creazione di nuovi componenti UI in un progetto Astro.
+
+Volevamo anche assicurarci che Astro avesse un ottimo linguaggio di componenti incorporato. Per farlo, abbiamo creato il nostro linguaggio di interfaccia utente `.astro`. È fortemente influenzato da HTML: qualsiasi frammento valido di HTML è già un componente Astro valido! Ma combina anche alcune delle nostre funzionalità preferite prese in prestito da altri linguaggi di componenti, come le espressioni JSX (React) e l'ambito CSS per impostazione predefinita (Svelte e Vue). Questa somiglianza con HTML semplifica anche l'utilizzo di miglioramenti progressivi e modelli di accessibilità comuni senza alcun sovraccarico.
+
+Astro è stato progettato per essere meno complesso rispetto ad altri framework e linguaggi di interfaccia utente. Uno dei motivi principali è che Astro è stato progettato per il rendering sul server, non nel browser. Ciò significa che non devi preoccuparti di: hooks (React), chiusure obsolete (anche React), riferimenti (Vue), osservabili (Svelte), atomi, selettori, reazioni o derivate. Non c'è reattività sul server, quindi tutta quella complessità svanisce.
```

Comment:
- I would leave some terms in English here, since they are very specific and refer to very specific concept in JavaScript (maybe we can use italic for them): 
  - "stale closures"
  - "refs"
  - "observables"
  - not sure about "reactions" and "derivations": both the Italian translations remind me of something else (maths and chemistry).
 
- I would translate "(also React)" with "(di nuovo, React)"

Reply:
I applied the fixes!